Update Data_Dictionary_Release_Notes.md

zhangm-02 opened the pull request:
- Family_history.relationship_sex_at_birth changed to `new`
- Family_history.relationship_gender changed to `deprecated`
- Diagnosis properties removed from pathology_detail
- `morphology` moved to diagnosis
- Follow_up.days_to_first_event removed from changelist
- Removed analyte.analyte_type_id from changelist
- `After Study Enrollment` and `Prior to Study Enrollment` added to timepoint_category as deprecated
- 9385/3 and 9478/3 added as new morphology values
- Removed 'deprecated' from follow_up entity
- Added 'indeterminant' as a deprecated value to margin_status

zhangm-02 committed:
Update Data_Dictionary_Release_Notes.md

- Family_history.relationship_sex_at_birth changed to `new`
- Family_history.relationship_gender changed to `deprecated`
- Diagnosis properties removed from pathology_detail
- `morphology` moved to diagnosis
- Follow_up.days_to_first_event removed from changelist
- Removed analyte.analyte_type_id from changelist
- `After Study Enrollment` and `Prior to Study Enrollment` added to timepoint_category as deprecated
- 9385/3 and 9478/3 added as new morphology values
- Removed 'deprecated' from follow_up entity
- Added 'indeterminant' as a deprecated value to margin_status

wwysoc2 commented on docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:
```diff
@@ -49,9 +49,18 @@
 	* Changes made to `population_group`
 		* New permissible value: `Unknown`
 		* New permissible value: `Not Reported`
+* Altered `diagnosis` entity
```
I think something about  the following properties needs to be added here:  `metastasis_at_diagnosis_site, mitotic_count, pregnant_at_diagnosis, primary_disease`

zhangm-02 replied:
I didn't add those because I didn't see those being altered on the spreadsheet - I think the script might have thought they were new additions pathology_detail and marked them as new but nothing was actually changed. 

If you can, double check this, I might be wrong on that bit.

wwysoc2 reviewed (changes requested):
Add the other diagnosis fields, and we should be good 👍

zhangm-02 committed:
Update Data_Dictionary_Release_Notes.md

Added the four removed properties to diagnosis

zhangm-02 committed:
Update Data_Dictionary_Release_Notes.md

Moved `micropapillary_features`, `papillary_renal_cell_type` under diagnosis as removed
